run_uh.py

```diff
@@ -67,10 +67,9 @@ def exit_with_error(title, message):
 
 def check_python_version():
 	# python up to version 2.6.1 returns an int. http://bugs.python.org/issue5561
-	if platform.python_version_tuple()[0] not in (2,'2'):
-		exit_with_error('Unsupported Python version', 'Python 2 is required to run Unknown Horizons.')
-
-
+    if float(sys.version[:3]) < 2.7:
```

Comment:
I wasn't aware that sys.version_info can be sliced into a tuple. I agree that a tuple-to-tuple comparison is cheaper on memory, though I would write this instead:
`if sys.version_info[:2] != (2, 7):`
For now we assume Python 2.7 is the only fully supported version, and leaving it at "<" would not exclude Python 3.x which we don't support just yet :).